extensions/binding/org.eclipse.smarthome.binding.fsinternetradio/src/main/java/org/eclipse/smarthome/binding/fsinternetradio/internal/radio/FrontierSiliconRadioConnection.java

```diff
@@ -50,7 +50,7 @@
     private String sessionId;
 
     /** http clients, store cookies, so it is kept in connection class. */
-    private HttpClient httpClient = null;
+    private static final HttpClient HTTP_CLIENT = new HttpClient();
```

Comment:
Does it really have to be static? First of all, the javadoc does then not really fit anymore.
Secondly, I would expect that there is only one connection per radio - I see that you mentioned that new instances are created over and over again, but maybe this is then the issue that should be addressed?